aws/data_source_aws_glue_data_catalog_encryption_settings_test.go

```diff
@@ -0,0 +1,64 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/service/glue"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+func TestAccDataSourceAwsGlueDataCatalogEncryptionSettings_basic(t *testing.T) {
+	resourceName := "aws_glue_data_catalog_encryption_settings.test"
+	datasourceName := "data.aws_glue_data_catalog_encryption_settings.test"
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:   func() { testAccPreCheck(t) },
+		ErrorCheck: testAccErrorCheck(t, glue.EndpointsID),
+		Providers:  testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDataSourceAwsGlueDataCatalogEncryptionSettingsConfig(),
+				Check: resource.ComposeTestCheckFunc(
+					testAccDataSourceAwsGlueDataCatalogEncryptionSettingsCheck(datasourceName),
```

Comment:
from what i see in other datasource tests we don't need this check here. (the data source is not creating anything so there is nothing to check here)

Reply:
I based this from `data_source_aws_kms_key_test.go`. Could you point me to another file whose structure I should copy?